### PR TITLE
impl: Added settings command, for people who ask about settings

### PR DESCRIPTION
**Pull request by d1snin**

### CODE REVIEW

File: kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/SettingsCommand.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package org.kamiblue.botkt.plugins.main.commands
+
+import org.kamiblue.botkt.command.BotCommand
+import org.kamiblue.botkt.command.Category
+import org.kamiblue.botkt.utils.normal
+
+object SafeCommand : BotCommand(
+        name = "safe",
+        description = "Is KAMI Blue safe?",
```
Comment (5HT2) [inline kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/SettingsCommand.kt]: you forgot to change all this

File: kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/SettingsCommand.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        category = Category.INFO
+) {
+    init {
+        execute {
+            channel.normal(
+                    title = "Settings for KAMI Blue",
```
Comment (5HT2) [inline kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/SettingsCommand.kt]: change this to "CrystalAura settings"

File: kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/SettingsCommand.kt [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+                    description = "The default settings are already good, just adjust the settings to suit your playstyle and ping.")
```
Comment (5HT2) [inline kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/SettingsCommand.kt]: change this to "The default settings in the latest beta are usually good, adjust the settings to your play style and ping.\nIf you have used older versions of KAMI Blue then press the "Defaults" button to get the new default settings"

**Review by 5HT2 — CHANGES_REQUESTED** [review]

**Comment by d1snin** [comment]
im monkey

File: kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/SettingsCommand.kt [hunk 4]
```diff
@@ -0,0 +1,19 @@
+package org.kamiblue.botkt.plugins.main.commands
+
+import org.kamiblue.botkt.command.BotCommand
+import org.kamiblue.botkt.command.Category
+import org.kamiblue.botkt.utils.normal
+
+object SettingsCommand : BotCommand(
+        name = "settings",
```
Comment (5HT2) [inline kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/commands/SettingsCommand.kt]: change this to CASettingsCommand and the name to casettings

File: kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/KamiBluePlugin.kt [hunk 5]
```diff
@@ -51,6 +51,7 @@ class KamiBluePlugin : Plugin() {
         MonkeyCommand,
         RandomMonkeyCommand,
         SafeCommand,
+        SettingsCommand,
```
Comment (5HT2) [inline kami-blue-plugin/src/org/kamiblue/botkt/plugins/main/KamiBluePlugin.kt]: change this to CASettingsCommand

**Review by 5HT2 — CHANGES_REQUESTED** [review]

**Review by 5HT2 — APPROVED** [review]

**Pull request merged**